scripts/buf_lint_check.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+jar_file_path=$?
+config_file_path=$?
+os_type=$?
+github_actions_path=$1
+
+check_protobuf() {
+  echo "********************************"
+  echo "Checking Protobuf file formatting"
+  echo "********************************"
+
+  $jar_file_path lint --config $config_file_path
+
+  status=$?
+
+  if [ "$status" = 0 ] ; then
+    echo "Protobuf lint check completed successfully"
+    exit 0
+  else
+    echo "********************************"
+    echo "Protobuf lint check issues found"
+    echo "Please fix the above issues."
+    echo "********************************"
+    exit 1
+  fi
+}
+
+windows_warning() {
+  echo "********************************"
+  echo "Currently Buf check doesn't support on Windows"
+  echo "********************************"
+}
+
+create_jar_file_path() {
+  buf_file_name=$?
+
+  if [ "$1" = "Linux" ]; then
+    buf_file_name="buf-Linux-x86_64"
+  else
+    buf_file_name="buf-Darwin-x86_64"
+  fi
+
+  if [ $github_actions_path ]; then
+    jar_file_path="$github_actions_path/oppia-android-tools/$buf_file_name"
+    config_file_path="/home/runner/work/oppia-android/oppia-android/buf.yaml"
+  else
+    jar_file_path="../oppia-android-tools/$buf_file_name"
+    config_file_path="buf.yaml"
+  fi
+}
+
+check_github_action_path_exists() {
```

Comment:
This isn't checking, it's ensuring. Please update the name to be more specific to what the function is doing. E.g.: ensure_jar_config_file_paths_are_set

Reply:
Done